crates/project/src/direnv.rs

```diff
@@ -0,0 +1,72 @@
+use crate::environment::EnvironmentErrorMessage;
+use std::process::ExitStatus;
+
+#[cfg(not(any(test, feature = "test-support")))]
```

Comment:
Do you need to include this? I thought that `collections` does this on its own.

Reply:
The `use {collections::HashMap, std::path::Path, util::ResultExt};` is only used in the `load_direnv_environment` which is disabled when either `test` or `test-support` are enabled, so clippy complains about it if it's not behind a `cfg`-guard